src/ggrc/assets/javascripts/components/csv/export.js

```diff
@@ -158,16 +158,21 @@
             data_grid = this.scope.attr("export.data_grid"),
             only_relevant = this.scope.attr("export.only_relevant"),
             query = _.map(panels, function (panel, index) {
-              var relevant_filter = "";
+              var relevant_filter = "",
+                  predicates;
               if (data_grid && index > 0) {
                 relevant_filter = "#__previous__," + (index - 1) + "#";
                 if (only_relevant && index > 1) {
-                  relevant_filter += " AND #__previous__," + (index - 2) + "#";
+                  relevant_filter += " AND #__previous__,0#";
                 }
               } else {
-                relevant_filter = _.reduce(panel.relevant(), function (query, el) {
-                  return (query && query + " AND ") + "#" + el.model_name + "," + el.filter.id + "#";
-                }, "");
+                predicates = _.map(panel.relevant(), function (el) {
```

Comment:
I get the pannel with 
 var panels = this.scope.attr("export.panels.items"),
and map through that. so I think it should be fine.